pkg/proto/agent.proto

```diff
@@ -44,10 +48,11 @@ message CloseLabRequest {
     string labTag = 1;
 }
 
-message AddExercisesRequest {
+message ExerciseRequest {
     string labTag = 1;
     string envTag = 2;
     repeated string exercises = 3;
+    string exercise = 4;
```

Comment:
Why add another line? you could just send a single one in the exercises part ;P

Reply:
We discussed this in our meeting